--- a/arch/arm64/boot/dts/marvell/armada-3720-espressobin-ultra.dts	2023-10-01 20:02:25.657320292 -0700
+++ b/arch/arm64/boot/dts/marvell/armada-3720-espressobin-ultra.dts	2023-10-01 20:03:25.715059552 -0700
@@ -24,6 +24,8 @@
 		ethernet5 = &switch0port4;
 	};
 
+	/delete-node/ regulator;
+
 	reg_usb3_vbus: usb3-vbus {
 		compatible = "regulator-fixed";
 		regulator-name = "usb3-vbus";
@@ -66,6 +68,7 @@
 };
 
 &sdhci1 {
+	/delete-property/ vqmmc-supply;
 	status = "disabled";
 };
 
