--- a/drivers/cpufreq/armada-37xx-cpufreq.c	2024-01-13 21:45:22.923539098 -0800
+++ b/drivers/cpufreq/armada-37xx-cpufreq.c	2024-01-13 21:46:55.193294339 -0800
@@ -106,7 +106,7 @@
 	 * The cpufreq scaling for 1.2 GHz variant of the SOC is currently
 	 * unstable because we do not know how to configure it properly.
 	 */
-	/* {.cpu_freq_max = 1200*1000*1000, .divider = {1, 2, 4, 6} }, */
+	{.cpu_freq_max = 1200*1000*1000, .divider = {1, 2, 4, 6} },
 	{.cpu_freq_max = 1000*1000*1000, .divider = {1, 2, 4, 5} },
 	{.cpu_freq_max = 800*1000*1000,  .divider = {1, 2, 3, 4} },
 	{.cpu_freq_max = 600*1000*1000,  .divider = {2, 4, 5, 6} },
